### PR TITLE
Python3: Make some files compatible with Python3

### PR DESCRIPTION
**Pull request by nsoranzo**
Also enlarge the set of files tested with flake8 under Python3.

xref: #1715

### CODE REVIEW

**Comment by nsoranzo** [comment]
@galaxybot test this

File: contrib/nagios/check_galaxy.py [hunk 1]
```diff
@@ -75,15 +76,15 @@ def usage():
...
+            print("Specified -n, will create a new history")
         new_history = True
     else:
         usage()
 
 # state information
 var_dir = os.path.join( os.path.expanduser('~'), ".check_galaxy", server )
 if not os.access( var_dir, os.F_OK ):
-    os.makedirs( var_dir, 0700 )
+    os.makedirs( var_dir, 0o700 )
```
Comment (martenson) [inline contrib/nagios/check_galaxy.py]: Is `0o700` supported by 2.7? I did not find it mentioned.

**Comment by martenson** [comment]
lgtm, thanks @nsoranzo !

**Comment by nsoranzo** [comment]
@martenson Thanks for the review and merge!

**Pull request merged**